### PR TITLE
changed the container name in docker stats page

### PR DESCRIPTION
**Pull request by aslamahemad**
**- What I did**
- I changed the container name from drunk_visvesraya to humble_visvesraya. It "fixes #16720"

**- How I did it**
- $ sed -i 's/drunk/humble/g' stats.md

**- How to verify it**
- $ cat ~/cli/docs/reference/commandline/stats.md | grep "humble"

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by cpuguy83 — APPROVED** [review]

File: docs/reference/commandline/stats.md [hunk 1]
```diff
@@ -95,16 +95,16 @@ $ docker stats nginx --no-stream --format "{{ json . }}"
...
 
 ```console
-$ docker stats --all --format "table {{.Container}}\t{{.CPUPerc}}\t{{.MemUsage}}" fervent_panini 5acfcb1b4fd1 drunk_visvesvaraya big_heisenberg
+$ docker stats --all --format "table {{.Container}}\t{{.CPUPerc}}\t{{.MemUsage}}" fervent_panini 5acfcb1b4fd1 humble_visvesvaraya big_heisenberg
 
 CONTAINER                CPU %               MEM USAGE / LIMIT
 fervent_panini           0.00%               56KiB / 15.57GiB
 5acfcb1b4fd1             0.07%               32.86MiB / 15.57GiB
-drunk_visvesvaraya       0.00%               0B / 0B
+humble_visvesvaraya       0.00%               0B / 0B
```
Comment (thaJeztah) [inline docs/reference/commandline/stats.md:103]: We should probably fix the column aligning now (as the new name is 1 character wider)

**Review by thaJeztah — APPROVED** [review]
changing the name LGTM (thanks!)

looks like the PR may need a squash and a tiny fix-up for the column. Let me try fix that and push to this PR

**Comment by thaJeztah** [comment]
didn't want to force-push to a master branch, so I'm carrying this in https://github.com/docker/cli/pull/4031

**Comment by thaJeztah** [comment]
Thanks @aslamahemad ! Your changes were merged as part of https://github.com/docker/cli/pull/4031 (they still need to be backported and updated in the docs, so won't be "live" yet on docs.docker.com)

**Pull request closed without merging**